components/VerifiablePresentationRequest.yml

```diff
@@ -0,0 +1,81 @@
+openapi: 3.0.0
+info:
+  version: "0.0.3-unstable"
+  title: VC HTTP API
+  description: This is an Experimental Open API Specification for the [VC Data Model](https://www.w3.org/TR/vc-data-model/).
+  license:
+    name: W3C Software and Document License
+    url: http://www.w3.org/Consortium/Legal/copyright-software.
+  contact:
+    name: GitHub Source Code
+    url: https://github.com/w3c-ccg/vc-api
+paths:
+components:
+  schemas:
+    VerifiablePresentationRequest:
+      type: object
```

Comment:
any required fields here? or are they all optional?

Reply:
At present, all optional. There could be an argument to make at least "query" required. Raised an issue to track this concern: https://github.com/w3c-ccg/vp-request-spec/issues/16